#goal definition
string action_name
---
#result definition
geometry_msgs/PoseStamped grasp_candidate_camera_link
geometry_msgs/PoseStamped[] grasp_candidate_robot_link
geometry_msgs/PoseStamped[] approach_pose_robot_link
---
#feedback
string feedback

